Python/minestat/__init__.py

```diff
@@ -144,23 +147,30 @@ def __str__(self) -> str:
   """
 
 class MineStat:
-  VERSION = "2.4.3"             # MineStat version
+  VERSION = "2.5.0"             # MineStat version
   DEFAULT_TCP_PORT = 25565      # default TCP port for SLP queries
   DEFAULT_BEDROCK_PORT = 19132  # default UDP port for Bedrock/MCPE servers
   DEFAULT_TIMEOUT = 5           # default TCP timeout in seconds
 
   def __init__(self, address: str, port: int = 0, timeout: int = DEFAULT_TIMEOUT, query_protocol: SlpProtocols = SlpProtocols.ALL) -> None:
     self.address: str = address
     """hostname or IP address of the Minecraft server"""
+
     autoport: bool = False
     if port == 0:
       autoport = True
+
       if query_protocol is SlpProtocols.BEDROCK_RAKNET:
-        self.port = self.DEFAULT_BEDROCK_PORT
+        port = self._resolve_srv_record(self.address, proto="udp")
```

Comment:
There is currently no support for SRV records in Minecraft Bedrock: https://feedback.minecraft.net/hc/en-us/community/posts/360016429012-SRV-Record-Support-Bedrock-Edition-